coala_use.md

```diff
@@ -0,0 +1,29 @@
+#Coala
+
+This is a guide on how to use coala-bears for this project
+
+This project consists of a .coafile which basically has all the guidelines
```

Comment:
.coafile should be in codespace like `.coafile`

Reply:
.coafile is there in the codespace. This is just the documentation